README.md

```diff
@@ -60,66 +69,6 @@ To be developed:
-      - Add the line: `"[tool]": ".[tool].fre[tool].[tool]Cli"`
-      - (Recommended): If the update is significant, consider incrementing the version number within [`setup.py`](https://github.com/NOAA-GFDL/fre-cli/blob/088ad363392b3bf187119d8970c22779d59aaed0/setup.py#L5) to reflect and signify the changes 
-  10. Test by running `pip install .` from the root level of the directory, and running `fre`, followed by any subcommands necessary
- 
-### **Adding Tools From Other Repositories**
-
-* Currently, the solution to this task is to approach it using Conda packages. The tool that is being added must reside within a repository that contains a meta.yaml that includes Conda dependencies like the one in this repository and ideally a setup.py (may be subject to change due to deprecation) that may include any potentially needed pip dependencies
-    - Once published as a Conda package, ideally on the [NOAA-GFDL channel](https://anaconda.org/NOAA-GFDL), an addition can be made to the "run" section under the "requirements" category in the meta.yaml of the fre-cli following the syntax `channel::package`
-    - On pushes to the main branch, the [package](https://anaconda.org/NOAA-GFDL/fre-cli) will automatically be updated using the workflow file
- 
-### **MANIFEST.in**
-
-* In the case where non-python files like templates, examples, and outputs are to be included in the fre-cli package, MANIFEST.in can provide the solution. Ensure that the file exists within the correct folder, and add a line to the MANIFEST.in following [this syntax](https://setuptools.pypa.io/en/latest/userguide/miscellaneous.html)
-    - For more efficiency, if there are multiple files of the same type needed, the MANIFEST.in addition can be something like `recursive-include fre/fre(subTool) *.fileExtension` which would recursively include every file matching that fileExtension within the specified directory and its respective subdirectories. Currently, fre-cli recursively includes every python and non-python file inside of /fre, although this may change in the future
-    - `setup.py` handles these files using [setuptools and namespace package finding](https://setuptools.pypa.io/en/latest/userguide/package_discovery.html)
-
-### **Example /fre Directory Structure**
-```
-.
-├── __init__.py
-├── fre.py
-├── README-tool-template.md
-├── lazy_group.py
-├── /[tool]
-│   ├── __init__.py
-│   ├── fre[tool].py
-│   ├── README.md
-│   ├── [subCommandScript].py
-│   └── /[optional-submodule]
-```
-
-## **Additional Helpful Links**
-* [Official Click Documentation](https://click.palletsprojects.com/en/8.1.x/api/)
-* [`setup.py` Key Words](https://setuptools.pypa.io/en/latest/references/keywords.html)
-* [`meta.yaml` Documentation](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html)
+## ** Contributing to fre-cli **
+
+See [`CONTRIBUTING.md`](https://github.com/NOAA-GFDL/fre-cli/blob/breakup_README/CONTRIBUTING.md)
```

Comment:
I think maybe we should put this section above `Further Usage notes`, to keep development kind of together

Reply:
that way, users can see how to initialize/start usage (with the methods) and then go into contributing, if they wanted

Reply:
good catch, changing...

Reply:
How are we looking now?